gcloud/language/sentiment.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Definition for Google Cloud Natural Language API sentiment.
+
+Sentiment is the response to an ``analyzeSentiment`` request.
+"""
+
+
+class Sentiment(object):
+    """A Google Cloud Natural Language API sentiment object.
+
+    .. _Sentiment message: https://cloud.google.com/natural-language/\
+                           reference/rest/v1beta1/Sentiment
+
+    See `Sentiment message`_.
+
+    :type polarity: float
+    :param polarity: Polarity of the sentiment in the ``[-1.0, 1.0]`` range.
+                     Larger numbers represent more positive sentiments.
```

Comment:
Sorry - was on vacation last week.

If you want, you can play around with the bigquery table [nl-wikipedia:nl_wikipedia.nl_wikipedia](https://bigquery.cloud.google.com/table/nl-wikipedia:nl_wikipedia.nl_wikipedia?pli=1), which ran sentiment on all wikipedia articles. Playing around with that, it confirms that polarity isn't necessarily just -1, 0, or 1.

As @tseaver mentioned, polarity and magnitude both provide insight into the sentiment of a text blob. Polarity gives the overall negativity / positivity, and magnitude gives a sense of how emotional in general the text blob was. Conceptually, magnitude is roughly `sum(abs(sentence.polarity) for sentence in text_blob)`. Side note - magnitude scales with the length of the text, whereas polarity doesn't.